orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/MonitorWebhookTask.groovy

```diff
@@ -53,48 +53,54 @@ class MonitorWebhookTask implements OverridableTimeoutRetryableTask {
   @Autowired
   WebhookService webhookService
 
-  static requiredParameters = ["statusEndpoint", "statusJsonPath"]
-
   @Override
   TaskResult execute(Stage stage) {
-    def missing = requiredParameters.findAll { !stage.context.get(it) }
-    if (!missing.empty) {
-      throw new IllegalStateException("Missing required parameter${missing.size() > 1 ? 's' : ''} '${missing.join('\', \'')}'")
-    }
+    StageData stageData = stage.mapTo(StageData)
 
-    String statusEndpoint = stage.context.statusEndpoint
-    String statusJsonPath = stage.context.statusJsonPath
-    String progressJsonPath = stage.context.progressJsonPath
-    String successStatuses = stage.context.successStatuses
-    String canceledStatuses = stage.context.canceledStatuses
-    String terminalStatuses = stage.context.terminalStatuses
-    def customHeaders = stage.context.customHeaders
+    if (stageData.statusEndpoint == null || stageData.statusJsonPath == null) {
```

Comment:
Has this changed the behavior if either `statusEndpoint` or `statusJsonPath` is the empty string?  I think before it would throw an exception before where as now it will not.

Reply:
yes, thanks!